__tests__/app.test.js

```diff
@@ -22,4 +22,40 @@ describe("NC_Games API", () => {
         });
     });
   });
+  describe("GET /api/reviews/:review_id", () => {
+    it("200: Returns all reviews for the given review_id", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({ body }) => {
+          const { review } = body;
+          expect(review).toHaveProperty("review_id", expect.any(Number));
```

Comment:
At the moment, you're not actually checking you've got the correct review back. 
This would pass if the wrong review came back. 
You need to check you are getting the correct id